### ISSUE TITLE
Print out settings when run

### ISSUE THREAD

**Issue by pipitone** [pipitone/qbatch#76]
I wonder if it might be helpful to have a tiny display of the actual settings used (just in case users don't realize what the defaults are, etc), e.g. 

```
$ qbatch -w 1:00:00 commands.txt
Processors requested per job (--ppj): 8
      Processes run in parallel (-j): 1
               Commands per job (-c): 8
                       Walltime (-w): 1:00:00 
                       Other options: none
```

Thoughts?

**Comment by gdevenyi** [comment]
Yes, that looks good, however maybe it should only do so under verbose?

**Comment by gdevenyi** [comment]
Followup here, I like the initial implementation, but should it be more verbose as to settings, and should those settings only be printed if relevant? i.e. in local mode, certain things suppressed?

**Comment by gdevenyi** [comment]
If... we were to implement a config file, this could print out the internal config state.

**Comment by pipitone** [comment]
Sure... but I don't see how the config file has anything to do with this
because this issue is about displaying the configuration qbatch is in
(because of some base config plus command line options) when your jobs
were submitted, right?

**Comment by gdevenyi** [comment]
Right, I was just thinking we could hijack the configparser saving features to print to screen, no custom formatting or such required, would also allow a copy-paste to save that config (if we go with that).

**Comment by pipitone** [comment]
Yeah, that's a nice benefit: config display that doubles as a usable
config file. As long as it doesn't get in the way of actually being
readable/meaningful for the humans. :-)

**Comment by gdevenyi** [comment]
Looks like configparser config files are nice and readable:
http://effbot.org/librarybook/configparser.htm

**Comment by gdevenyi** [comment]
Merged into #96
